GetComponents

```diff
@@ -385,7 +384,7 @@ sub parse_list {
     }
     # if $ROOT is undefined, it will be set to the current directory
     if   ( defined( $DEFINITIONS{ROOT} ) ) { $ROOT = $DEFINITIONS{ROOT} }
-    else                                   { $ROOT = $orig_dir }
+    else                                   { $ROOT = cwd() }
```

Comment:
Wouldn't this be a problem as well in several places later, if cwd() contains spaces (see also comment below)?